.travis.yml

```diff
@@ -1,20 +1,24 @@
----
 language: ruby
 bundler_args: "--without development"
 sudo: false
 cache: bundler
 env:
   global:
-    - GIT_COMMITTED_AT=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then git log -1 --pretty=format:%ct; else git log -1 --skip 1 --pretty=format:%ct; fi)
+  - CC_TEST_REPORTER_ID=$CODECLIMATE_REPO_TOKEN
+  - GIT_COMMITTED_AT=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then git log -1 --pretty=format:%ct;
+    else git log -1 --skip 1 --pretty=format:%ct; fi)
 rvm:
-  - 2.3.3
+- 2.3.3
 before_script:
-  - gem install  --no-rdoc --no-ri 'brakeman' 'bundler-audit'
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - RAILS_ENV=test bundle exec rake db:create db:schema:load
+- gem install  --no-rdoc --no-ri 'brakeman' 'bundler-audit'
+- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+- chmod +x ./cc-test-reporter
+- RAILS_ENV=test bundle exec rake db:create db:schema:load
+addons:
+  code_climate:
+    repo_token: 50908698e3f680091760f1b1d09e447736d9eb973a44ce91bd3ff6aa7a8c929e
```

Comment:
Does this need to be here? There's `$CODECLIMATE_REPO_TOKEN` already set in the env on travis.
Does it need it set in both places?